src/components/EventCard/styles.js

```diff
@@ -37,7 +37,7 @@ const styles = (theme) => ({
   eventCardName: {
     marginBottom: theme.padding.xs.px,
     display: '-webkit-box',
-    maxWidth: '400px',
+    width: '400px',
```

Comment:
try adding `flex: 1` to this, and make sure the parent container has `display: flex`. read about using flexbox. flex: 1 will take up all the space of the parent container.

this solution of setting a width is not good for responsive design.